Tests/Indicators/ForceIndexTests.cs

```diff
@@ -0,0 +1,38 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using NUnit.Framework;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Tests.Indicators
+{
+    [TestFixture]
+    public class ForceIndexTests : CommonIndicatorTests<TradeBar>
```

Comment:
Some tests seems to be failing
<img width="326" alt="image" src="https://github.com/QuantConnect/Lean/assets/18473240/3a6df521-29ea-4da2-9d22-8420f86a96c5">

Reply:
Well this one is more complicated and will probably need some investigation from your side.

I encountered similar problem with some others PR.

That's why you you see https://github.com/QuantConnect/Lean/pulls/femtotrader Tasks 5 of 11 and not 6 of 11

My guess is that either I didn't understand what happen when indicator have Renko Bars as input (if this kind of indicator can really take them) or if I should short-circuit test or if I need to see if the problem is on AcceptsRenkoBarsAsInput code?

Anyway some help / guidance will help as it's locking

Reply:
Suggest debug the failing test and underlying why it's failing, some of these method in the base indicator test suite are virtual so they can be overwritten if required, u can see there are other couple of implementations of AcceptsRenkoBarsAsInput. But first should validate/understand what's going on, to know what's the solution 👍

Reply:
I won't be available more today neither tomorrow. Feel free to take the lead on this one (it should help others waiting PR)